vo_new/components/dashboard/india-map-chart.tsx

```
@@ -0,0 +1,977 @@
+
+      for (const source of geoJsonSources) {
+        try {
+          const response = await fetch(source)
+          if (response.ok) {
+            geoJson = await response.json()
+            sourceUsed = source
+            break
+          }
+        } catch (err) {
+          console.warn(`Failed to load from ${source}:`, err)
+          continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)